README.md

```diff
@@ -181,9 +198,9 @@ Furthermore, when developing/debugging a new scenario:
 
  1. Add a `@wip` tag decorator on the scenario
  2. To only run @wip scenarios run: `tox -e behave-20.04 -- -w`
- 4. If you want to use a debugger:
-    a. Add ipdb to integration-requirements.txt
-    b. Add ipdb.set_trace() in the code block you wish to debug
+ 3. If you want to use a debugger:
+    1. Add ipdb to integration-requirements.txt
+    2. Add ipdb.set_trace() in the code block you wish to debug
```

Comment:
The letters here weren't rendering properly. I prefer the look of the letters in plaintext, but when rendered as html, the sub-ordered list looks better to me. LMK if you'd rather I change it back